### DISCUSSION TITLE
button search openai/gemini froma specified image on 0.15

### DISCUSSION THREAD

**Discussion by divemasterjm** [blakeblackshear/frigate#13906] [Ideas]
Sometimes i found some images without description, could some connection/max petitions. etc.
would be nice to have an aditional button on images to do a recall to gemini/openai for this specific image that has no description.

**Comment by hawkeye217** [comment]
That's a good idea. I've implemented it in https://github.com/blakeblackshear/frigate/pull/13930

**Reply by divemasterjm** [reply]
thanks!!!
